packages/yoastseo/src/values/Paper.js

```diff
@@ -17,24 +17,27 @@ const defaultAttributes = {
 	date: "",
 	customData: {},
 	textTitle: "",
+	isRTL: false,
 };
 
 /**
  * Constructs the Paper object and sets the keyword property.
  *
- * @param {string} text                     The text to use in the analysis.
- * @param {object} [attributes]             The object containing all attributes.
- * @param {string} [attributes.keyword]     The main keyword.
- * @param {string} [attributes.synonyms]    The main keyword's synonyms.
- * @param {string} [attributes.description] The SEO description.
- * @param {string} [attributes.title]       The SEO title.
- * @param {number} [attributes.titleWidth]  The width of the title in pixels.
- * @param {string} [attributes.slug]        The slug.
- * @param {string} [attributes.locale]      The locale.
- * @param {string} [attributes.permalink]   The base url + slug.
- * @param {string} [attributes.date]        The date.
- * @param {Object} [attributes.wpBlocks]    The text, encoded in WordPress block editor blocks.
- * @param {Object} [attributes.customData]  Custom data.
+ * @param {string}  text                     The text to use in the analysis.
+ * @param {object}  [attributes]             The object containing all attributes.
+ * @param {string}  [attributes.keyword]     The main keyword.
+ * @param {string}  [attributes.synonyms]    The main keyword's synonyms.
+ * @param {string}  [attributes.description] The SEO description.
+ * @param {string}  [attributes.title]       The SEO title.
+ * @param {number}  [attributes.titleWidth]  The width of the title in pixels.
+ * @param {string}  [attributes.slug]        The slug.
+ * @param {string}  [attributes.locale]      The locale.
+ * @param {string}  [attributes.permalink]   The base url + slug.
+ * @param {string}  [attributes.date]        The date.
+ * @param {Object}  [attributes.wpBlocks]    The text, encoded in WordPress block editor blocks.
+ * @param {Object}  [attributes.customData]  Custom data.
+ * @param {string}  [attributes.textTitle]   The title of the text.
+ * @param {boolean} [attributes.isRTL]       Whether the locale is an RTL.
```

Comment:
In general, I don't want to rely too much on boolean properties. 

In my opinion many boolean properties can be derived from a more general property. Like `isRed` can be derived from a `color` property that (IMHO) would be more flexible and contain more information.

Reply:
I think I agree with you, that in this case, we should just go with the non-boolean property. I think it would be also more informative that the property is called "writingDirection", we can always assign "ltr" as the default value. But at least we're also more inclusive in this case? as in we don't exclude people who potentially want to use yoastseo for a different writing direction other than LTR/RTL